dotnet/samples/01.messaging.echoBot/Nuget.Config

```diff
@@ -2,6 +2,14 @@
 <!-- Temporary local nuget source before public release -->
 <configuration>
 	<packageSources>
-		<add key="Local Package" value="./LocalPkg" />
+		<clear />
+        <add key="Local Package" value="./LocalPkg" />
+		<add key="github" value="https://nuget.pkg.github.com/microsoft/index.json" />
```

Comment:
It seems the indent is a bit weird. Is that because mixed tab ("/t") and space (" ")?

Reply:
good catch, ill fit that

Reply:
I think I resolved it, I also added a editoconfig to each sample to try and make sure it doesn't happen again